lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -244,6 +244,7 @@ namespace :foreman_tasks do
 
     tasks = ForemanTasks::Task.search_for(filter)
 
+    puts _("Exporting all tasks matching filter #{full_filter}")
```

Comment:
I'm getting `NameError: undefined local variable or method `full_filter' for main:Object` when testing this PR